upf/upf_pfcp_api.c

```diff
@@ -3101,12 +3102,14 @@ upf_pfcp_handle_msg (pfcp_msg_t *msg)
     {
       /* not enough info in the message to produce any meaningful reply */
       upf_debug ("PFCP: broken message");
+      upf_increment_counter (UPF_PFCP_RECEIVED_CORRUPTED, 0, 1);
       return -1;
     }
 
   if (r != 0) /* if cause != 0 */
     {
       upf_debug ("PFCP: error response %d", r);
+      upf_increment_counter (UPF_PFCP_RECEIVED_CORRUPTED, 0, 1);
```

Comment:
feels like we could use `UPF_PFCP_***_ERROR` here as well, since type is known and error response is sent